### PR TITLE
add polish translation

### PR DESCRIPTION
**Pull request by AngryPenguinPL**

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+msgstr "Ok"
+
+#: ../src/ui/WebView.cpp:77
+msgid "Cancel"
+msgstr "Anuluj"
+
+#: ../src/ui/WebView.cpp:328
+msgid "Unresponsive"
+msgstr "Nie odpowiada
+#: ../src/ui/WebView.cpp:329
```
Comment (eneshecan) [inline po/pl.po:59]: Could you add a newline after this? Probably, the build fails because of this

Comment (AngryPenguinPL) [reply po/pl.po:60]: Done

File: po/pl.po [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+msgid "Ok"
+msgstr "Ok"
+
+#: ../src/ui/WebView.cpp:77
+msgid "Cancel"
+msgstr "Anuluj"
+
+#: ../src/ui/WebView.cpp:328
+msgid "Unresponsive"
+msgstr "Nie odpowiada
```
Comment (eneshecan) [inline po/pl.po:58]: Ah, missing "

**Review by eneshecan — CHANGES_REQUESTED** [review]

File: po/pl.po (hunk 2 above)
Comment (AngryPenguinPL) [reply po/pl.po]: Should be done.

**Pull request merged**